File: auth-service/src/main/java/com/example/auth_service/dto/RegisterRequest.java
package com.example.auth_service.dto;

public class RegisterRequest {
    private String name;
    private String email;
    private String passwordHash;

    private String dietPreference;
    private String ageGroup;

    // getters & setters
    public String getName() { return name; }
    public void setName(String name) { this.name = name; }

    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }

    public String getPasswordHash() { return passwordHash; }
    public void setPasswordHash(String password) { this.passwordHash = password; }

    public String getDietPreference() { return dietPreference; }
    public void setDietPreference(String dietPreference) { this.dietPreference = dietPreference; }

    public String getAgeGroup() { return ageGroup; }
    public void setAgeGroup(String ageGroup) { this.ageGroup = ageGroup; }
}

File: backend-base/SmartRecipeFinder.API/Models/RegisterRequest.cs
namespace SmartRecipeFinder.API.Models
{
    public class RegisterRequest
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string PasswordHash { get; set; }
        public string AgeGroup { get; set; }
        public string DietPreference { get; set; }
    }

}
